src/assets/bsc/index.js

```diff
@@ -0,0 +1,25 @@
+import _ from 'lodash'
+import { isValidAddress, toChecksumAddress } from 'ethereumjs-util'
+import networkAsset from './network'
+import tokensData from './tokens.json'
+import { mapCurrencyConverters } from '../mapCurrencyConverters'
+
+const tokens = _.mapValues(tokensData, token => ({
+  ...token,
+  network: 'bsc',
+  safeConfirmations: networkAsset.safeConfirmations,
+  type: 'bep20',
```

Comment:
I think this should be still the "erc20" 
otherwise in each client you have to know what is their equivalent type